src/components/PersonHoverCard/index.tsx

```diff
@@ -0,0 +1,117 @@
+import MailIcon from '@material-ui/icons/Mail';
+import PhoneIcon from '@material-ui/icons/Phone';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { Box, Card, Popover, Tooltip, Typography } from '@material-ui/core';
+
+import ZetkinPerson from 'components/ZetkinPerson';
+import { ZetkinTag } from 'types/zetkin';
+import { personResource, personTagsResource } from 'api/people';
+
+const TagChip: React.FunctionComponent<{ tag: ZetkinTag }> = ({ tag }) => {
+  return (
+    <Tooltip arrow title={tag.description}>
+      <Box
+        bgcolor={tag.color || '#e1e1e1'}
+        borderRadius="18px"
+        fontSize={13}
+        px={2}
+        py={0.7}
+      >
+        {tag.title}
+      </Box>
+    </Tooltip>
+  );
+};
+
+const PersonHoverCard: React.FunctionComponent<{ personId: string }> = ({
```

Comment:
The type of `personId` should be `number` I think, because that's the way it's typed in the base data.

Sometimes an ID will be a string, namely when it's read from the URL of a requested page. But the actual ID of a piece of data is always a number, so if we're going to convert one way, I think it makes more sense to convert/type cast _to number_.